<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGame</class>
 <widget class="QWidget" name="FormGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetButtons" native="true">
   <property name="geometry">
    <rect>
     <x>664</x>
     <y>240</y>
     <width>129</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>分数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelScore">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="buttonPause">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/assets/button.png);
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="buttonSave">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/assets/button.png);
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="buttonLoad">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/assets/button.png);
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>加载</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="buttonSetting">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/assets/button.png);
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="buttonExit">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/assets/button.png);
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelPause">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>272</y>
     <width>281</width>
     <height>73</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/assets/mask_black.png);
font: 20pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
